python/feathub/feature_tables/sinks/redis_sink.py

```diff
@@ -41,7 +42,16 @@ def __init__(
         :param namespace: The namespace to persist features in Redis. Feature tables
                           sinking to Redis sinks with different namespaces can save
                           records with the same key into Redis without overwriting
-                          each other.
+                          each other. Only valid when key_expr is not set.
+        :param key_expr: A string that represents a FeatHub expression which evaluates
```

Comment:
Can we additionally require and validate that the `key_expr` always contains the `__NAMESPACE__`,  `__FEATURE_NAME__`, and all primary key fields?